Adding ability to break down nether brick block types

Batboy272 opened the pull request.

Batboy272 committed:
Adding ability to break down nether brick block types

epolixa commented:
This looks good for the most part, however I would change the value of the 'group' property to 'nether_brick' instead of 'brick', since we do not want these recipes overlapping with the brick recipes.

Batboy272 committed:
Added the ability to breakd down nether brick block types.

You are now able to use the stone cutter to break down, Nether Brick Fences, slabs, stairs and walls, into Nether Brick (item)

epolixa commented:
Looks good! I'll merge this now
